packages/example-getting-started/src/controllers/todo.controller.ts

```diff
@@ -0,0 +1,63 @@
+// Copyright IBM Corp. 2017,2018. All Rights Reserved.
+// Node module: @loopback/example-getting-started
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {Filter, Where} from '@loopback/repository';
+import {post, param, get, put, patch, del} from '@loopback/openapi-v2';
+import {inject} from '@loopback/context';
+import {Todo} from '../models';
+import {TodoRepository} from '../repositories';
+
+export class TodoController {
+  constructor(
+    @inject('repositories.TodoRepository')
+    public todoRepository: TodoRepository,
+  ) {}
+
+  @post('/todo')
+  async create(@param.body('obj') obj: Todo): Promise<Todo> {
+    return await this.todoRepository.create(obj);
```

Comment:
Yesterday you mentioned that this pull request is not only copying over files from loopback4-example-getting-started repository, but also making changes in the actual implementation. That's really bad, how are we supposed to find the differences and review those changes? 

The code in the original repository contains many important TODO comments that we need to address for our MVP milestone. 

In this particular place, the original `createTodo` method has additional hand-written validation as a workaround for missing https://github.com/strongloop/loopback-next/issues/118. This validation should be preserved, or at least we should discuss whether and how we want to change this implementation bit.

Please update this pull request to copy as many source files in verbatim as is possible. Additional changes/updates/refactorings should be made in follow-up pull requests, only once the current codebase has been added to monorepo with as little modifications as possible.